apps/api/src/app/events/events.controller.ts

```diff
@@ -204,4 +208,32 @@ export class EventsController {
       })
     );
   }
+
+  @ExternalApiAccessible()
+  @UserAuthentication()
+  @ThrottlerCost(ApiRateLimitCostEnum.BULK)
+  @Post('/trigger/bulk-cancel')
+  @ApiOkResponse({
+    type: TriggerBulkCancelResponseDto,
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** The new endpoint takes an array of `transactionId`s, therefore the endpoint should product an array of cancel response Dtos, one response Dto per `transactionId`. We can achieve specification of this with Nestjs swagger plugin by wrapping the `type` field with square braces.‏
```suggestion
    type: [TriggerBulkCancelResponseDto],
```

Reply:
done